scripts/environment.sh

```diff
@@ -85,6 +85,13 @@ if [[ ${GINKGO_FOCUS:-} == "features" && ${GINKGO_SKIP:-} == "pivoting remediati
   export WORKER_MACHINE_COUNT=${WORKER_MACHINE_COUNT:-"2"}
 fi
 
+# K8s conformance test environment vars and config
+if [[ ${GINKGO_FOCUS:-} == "k8s-conformance" ]]; then
+  export NUM_NODES="6"
+  export CONTROL_PLANE_MACHINE_COUNT=${CONTROL_PLANE_MACHINE_COUNT:-"1"}
+  export WORKER_MACHINE_COUNT=${WORKER_MACHINE_COUNT:-"5"}
```

Comment:
Do we need so many? If we do, it would be good to link to some source to explain it. E.g. conformance tests need X nodes to test functionality Y...

Reply:
Thanks! Yes please, put a comment there with link to the CAPI test :slightly_smiling_face: